Document relationship (#59)
core-wg/yang-cbor: draft-ietf-core-sid.md

{{sid-file-format}} defines a standard file format used to store and publish
SIDs.

> For IETF managed YANG modules, the allocation of SIDs via the IANA mechanism specified in this document is MANDATORY.

Reviewer: I am a bit worried that _MANDATORY_ here might face some opposition (what about old ones), although it will probably make everybody's life easier. I believe the general idea so far was allocating SIDs at module creation or when needed.

Author: I think that the sentence is pretty clear.  If you want to allocate SIDs, AND it's an IETF yang module, then you have to go to IANA.  (I thought that MANDATORY was a synonym for MUST, but BCP14 disagrees.  I will rewrite to say MUST).
If old modules don't need SID values, then they won't get them.

Reviewer 2: In my mind, we are straddling the boundary between the two documents (yang-cbor, sid) here again.

> For IETF managed YANG modules, the allocation of SIDs via the IANA mechanism specified in this document is MANDATORY.

Author: ```suggestion
IETF managed YANG modules which need to allocate SIDs, MUST use the IANA mechanism specified in this document.
Other documents MAY use another allocation method as defined by those organizations, or MAY ask IANA to do the allocation.
```

For YANG modules created by other parties, the use of IANA allocation mechanisms via use of Mega-Ranges (see {{mega-range-registry}}) is RECOMMENDED.

> These other parties are free to make up their own mechanism, as is their perogative.

Author: ```suggestion
Within the Mega-Range allocation, other parties are free to make up their own mechanism.
```
